helm/primehub/templates/post-install.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{.Release.Name}}-bootstrap"
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/managed-by: {{.Release.Service | quote }}
+    app.kubernetes.io/instance: {{.Release.Name | quote }}
+    helm.sh/chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+  annotations:
+    # This is what defines this resource as a hook. Without this line, the
+    # job is considered part of the release.
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: "{{.Release.Name}}-bootstrap"
+      labels:
+        app.kubernetes.io/managed-by: {{.Release.Service | quote }}
+        app.kubernetes.io/instance: {{.Release.Name | quote }}
+        helm.sh/chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+    spec:
+      restartPolicy: Never
+      serviceAccountName: primehub-bootstrap-sa
+      initContainers:
+        - name: wait-for-keycloak
+          image: busybox
+          command:
+            - sh
+            - -c
+            - |
+              until printf "." && nc -z -w 2 {{ .Chart.Name }}-keycloak-http 80; do
+                  sleep 2;
+              done;
+              echo 'Keycloak OK ✓'
+      containers:
+        - name: primehub-keycloak-bootstrap
+          image: gcr.io/google-containers/hyperkube-amd64:v1.14.0
+          env:
+          - name: PRIMEHUB_NAMESPACE
+            value: {{ .Release.Namespace }}
+          - name: KEYCLOAK_NAMESPACE
+            value: {{ .Release.Namespace }}
+          - name: KEYCLOAK_NAME
+            value: {{ .Release.Name }}-keycloak-0
+          - name: KEYCLOAK_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Release.Name }}-keycloak-http
+                key: password
+          - name: PRIMEHUB_DOMAIN
+            value: {{ .Values.primehub.domain }}
+          - name: PRIMEHUB_ADMIN
+            value: keycloak
+          - name: PRIMEHUB_ADMIN_PASSWORD
+            value: CHANGEKEYCLOAKPASSWORD
```

Comment:
this should come from secret

Reply:
keycloak stores the password in its secret, but username only provides in its values.yaml.